modules/3d-tiles/src/cesium-ion-loader.ts

```diff
@@ -18,7 +18,7 @@ async function preload(url, options = {}) {
 /**
  * Loader for 3D tiles from Cesium ION
  */
-export const CesiumIonLoader: LoaderWithParser = {
+export const CesiumIonLoader: LoaderWithParser<unknown, never, LoaderOptions> = {
```

Comment:
We should define (and export) a `CesiumIONLoaderOptions` and clear up the options mess below and make sure types are observed in the code.